bigquery_etl/glam/templates/latest_versions_v1.sql

```diff
@@ -1,36 +1,20 @@
-{{ header }}
-WITH extracted AS (
+  {{ header }}
+WITH
+  extracted AS (
   SELECT
-    client_id,
-    channel,
-    app_version
+    build.`target`.channel AS channel,
+    MAX(mozfun.norm.extract_version(build.`target`.version,
+        'major')) AS latest_version,
   FROM
-    {{ source_table }}
+    `moz-fx-data-shared-prod.telemetry.buildhub2`
   WHERE
-    submission_date
-    BETWEEN DATE_SUB(@submission_date, INTERVAL 28 DAY)
-    AND @submission_date
-    AND channel IS NOT NULL
-),
-transformed AS (
-  SELECT
-    channel,
-    app_version
-  FROM
-    extracted
+    build.`source`.product = "firefox"
+    AND build.`target`.channel = {{ app_id_channel }}
+    AND DATE(build.build.date) <= @submission_date
   GROUP BY
-    channel,
-    app_version
-  HAVING
-    COUNT(DISTINCT client_id) > 5
```

Comment:
By getting rid of this condition we might allow some builds fewer than 5 users leak into the dataset.